### ISSUE TITLE
Slides option with more than one digit numbers does not work

### ISSUE THREAD

**Issue by utdrmac** [astefanutti/decktape#94]
If you specify `--slides` > 9, then decktape cannot parse this number properly and prints the wrong slide.

In this example, I only want to print slide 16. But what actually prints is slide 6.

```
[drmac@othala decktape-1.0.0]$ ./phantomjs decktape.js --slides 16 remark ~/Documents/tutorial.html ~/Documents/tutorial.pdf
Loading page /Users/drmac/Documents/tutorial.html ...
Loading page finished with status: success
Remark JS DeckTape plugin activated
Printing slide #6       (  6/103) ...
Printed 1 slides
```

Attempting 47 only printed slide 7, etc, etc

**Comment by utdrmac** [comment]
```
./phantomjs decktape.js --slides 15-17 ...
```

This printed slides 5 through 17 instead of only printing 3 slides, 15 16 & 17

**Comment by astefanutti** [comment]
Thanks a lot for the report. I've just fixed in @41861c1.

In the meantime a new release is cut, you can apply the change @41861c1 directly into you local installation.
